File: Dante1/Module1.04/Zadanie4.02/main.rs
fn main()
{
    let array:[f32; 16] = [10.0, 11.0, 4.0, 5.0, 6.0, 12.0, 16.0, 1.0, 
                            2.0, 3.0, 13.0, 14.0, 15.0, 7.0, 8.0, 9.0];

    for i in (0..array.len()).step_by(2) 
    {
        print!("{} ", array[i]);
    }
    
    return;
}

File: Dante1/Module1.04/Zadanie4.02/main.zig
const print = @import("std").debug.print;

pub fn main() !void {
    const array = [16]f32{ 10, 11, 4, 5, 6, 12, 16, 1, 2, 3, 13, 14, 15, 7, 8, 9 };
    for (&array) |i| {
        print("{d} ", .{i});
    }
}
